drizzle-orm/src/sqlite-core/README.md

```diff
@@ -264,6 +265,7 @@ export const cities = sqliteTable('cities', {
   name: text('name', { length: 256 }),
   countryId: integer('country_id').references(() => countries.id), // inline foreign key
   countryName: text('country_id'),
+  sisterCityId: integer('sister_city_id').references((): AnySQLiteColumn => cities.id), // self-referencing foreign key
```

Comment:
Could you also add the same example to Postgres and MySQL docs?

Reply:
Sure thing. I wanted to do this originally but haven't actually tested it in another other than SQLite. Assuming it works?